src/indexers/vault-harvests-indexer.ts

```diff
@@ -12,52 +12,54 @@ export async function updateVaultHarvests() {
     const currentBlock = await sdk.provider.getBlockNumber();
     const vaults = await chain.vaults.all();
 
-    for (const vault of vaults) {
-      try {
-        const { name, lastHarvestIndexedBlock } = vault;
+    await Promise.all(
```

Comment:
this will reject if one of the promises fails inside right? is that what we want

Reply:
yes but the entirety of the promise is in a try catch, so it will fail singularly and return as a success